rules.txt

```diff
@@ -212,7 +212,7 @@
 /departments/department-of-public-property/green-buildings/ 301  /departments/office-of-sustainability/about/energy-office/compliance/green-buildings/
 /departments/department-of-public-health/contact-us/ 301	 /departments/department-of-public-health/about-us/contact-us/
 /departments/department-of-public-health/breastfeeding-guides-policies-and-fact-sheets/ 301  /services/mental-physical-health/get-health-care-and-support/family-and-childrens-health-care/get-breastfeeding-support/
-/revenue/contact-us/                      301 /departments/department-of-revenue/about/contact-us/
+/revenue/contact-us                       301 /departments/department-of-revenue/about/contact-us/
```

Comment:
Can you move this redirect down with the other revenue redirects in alphabetical order?

Reply:
@m-atia done!